File: URI/1174.js
let input = require ('fs').readFileSync('./dev/stdin/file.txt','utf8')
let = lines = input.split('\n')

let lista = [];

for (let i = 0;i < 100;i++){
    lista.push(+lines.shift());
}

for (let j in lista){
    if (lista[j] <= 10){
        console.log(`A[${j}] = ${(lista[j]).toFixed(1)}`);
    }
}

File: Python/1174.py
lista = []
for x in range(0,100):
    lista.append (float(input()))

for y in range(0,100):
    if lista[y] <= 10:
        print("A [%0d] ="%y,lista[y])

for y in range(0,100):
    x = input()
    if x <= 10:
        print('A [{}] = {:0.2f}'.format (y, x))
        y += 1